drivers/sector_alarm_central/device.js

```diff
@@ -316,7 +316,7 @@ class MyDevice extends Homey.Device {
   async setAlarmState(state, action) {
     this.homey.app.updateLog('Function setAlarmState start', 2);
     return new Promise((resolve, reject) => {
-      if (code || code === '') {
+      if (code === '') {
```

Comment:
If code is null or undefined we still want to reject with an error message.
I think it should be:
`if (!code || code === '') {`

Reply:
Yes of course there should be a check for null or undefined.  

```suggestion
     if (!code || code === '') {
```